gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -326,25 +446,46 @@ def get_combine_faf_resources(
     return combine_faf_pipeline
 
 
+def filter_gene_to_test(ht: hl.Table) -> hl.Table:
+    """
+    Filter to PCSK9 1:55039447-55064852 for testing.
+
+    :param ht: Table with frequency and FAF information.
+    :return: Table with frequency and FAF information of the filtered interval of a gene
+    """
+    return hl.filter_intervals(
+        ht,
+        [hl.parse_locus_interval("chr1:55039447-55064852", reference_genome="GRCh38")],
+    )
+
+
 def main(args):
     """Create combined FAF resource."""
     hl.init(log="/compute_combined_faf.log")
     test = args.test
     overwrite = args.overwrite
-    pops = list(POPS["v3"] & POPS["v4"])
+    pops = POPS["v3"] + POPS["v4"]
     faf_pops = [pop for pop in pops if pop not in POPS_TO_REMOVE_FOR_POPMAX]
     combine_faf_resources = get_combine_faf_resources(overwrite, test, args.public)
 
     try:
         if args.create_combined_frequency_table:
             res = combine_faf_resources.combined_frequency
             res.check_resource_existence()
-            if test:
-                exomes_ht = res.exomes_ht.ht()._filter_partitions(range(20))
-                genomes_ht = res.genomes_ht.ht()._filter_partitions(range(20))
+            exomes_ht = res.exomes_ht.ht()
+            genomes_ht = res.genomes_ht.ht()
 
-            exomes_ht = extract_freq_info(exomes_ht, pops, faf_pops, "genomes")
-            genomes_ht = extract_freq_info(genomes_ht, pops, faf_pops, "exomes")
+            # replace oth with remaining in freq_meta and freq_index_dict for genomes_ht
+            genomes_ht = replace_oth_with_remaining(genomes_ht)
+            if test:
+                # exomes_ht = res.exomes_freq_ht.ht()._filter_partitions(range(20))
+                # genomes_ht = res.genomes_freq_ht.ht()._filter_partitions(range(20))
+                # filter to PCSK9 1:55039447-55064852 for testing
+                exomes_ht = filter_gene_to_test(exomes_ht)
+                genomes_ht = filter_gene_to_test(genomes_ht)
```

Comment:
Generally, the test should come before any other work is done so you reduce the amount of data your handling immediately. I know the replace_oth_with_remaining is working on freq globals but its best practice to filter to test things once you read your tables in. It would be better to keep the test and filter partitions and then add in another command line arg that will filter to a gene so we can test in different ways.